src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        return Color.values()[new Random().nextInt(Color.values().length)];
```

Comment:
make Random class-level variable, divide yout code to separate lines
```suggestion
    public String getRandomColor() {
        return Color.values()[new Random().nextInt(Color.values().length)];
```

Reply:
Thanks, fixed :)